main/ertp/guide/create-specific-erights.md

```diff
@@ -0,0 +1,87 @@
+# Creating a specific eright with ERTP
+
+## Definition of "specific"
+
+In the context of ERTP and assets, "specific" means "non-fungible".
+
+Assets of the same assay are not fungible if each different item of the assay
+cannot be replaced with another.
+House ownership rights are not fungible, because each ownership right refers to a 
+specific house and one house is not equal to another.
+
+## Creating a fungible asset with ERTP
+
+### Creating and sending the asset
+
+In ERTP, digital assets are created by a [`mint`](./mint.html). Having access to the mint
+gives you the power to create more digital assets of the same type at
+will.
+
+Let's say we own an Opera and want to sell tickets to seats for ballet shows. Tickets are
```

Comment:
I love this example!

Reply:
ahah
i wasn't sure this would be accepted, i'm glad this is your reaction ^^